t/app/controller/report_inspect.t

```diff
@@ -589,7 +593,27 @@ FixMyStreet::override_config {
         $mech->get_ok("/report/$report_id");
         $mech->content_contains('Nearest calculated address', 'Address displayed');
         $mech->content_contains('Constitution Hill, London, SW1A', 'Correct address displayed');
-    }
+    };
+
+    subtest "test upload photo with public updates" => sub {
+        $user->user_body_permissions->delete;
+        $user->user_body_permissions->create({ body => $oxon, permission_type => 'report_inspect' });
+
+        $report->state('confirmed');
+        $report->update;
+        $mech->get_ok("/report/$report_id");
+        $mech->submit_form_ok({ button => 'save', with_fields => {
+            public_update => "This is a public update.", include_update => "1",
+            state => 'action scheduled',
+            photo1 => [ [ $sample_file, undef, Content_Type => 'image/jpeg' ], 1 ],
+        } });
+        $report->discard_changes;
+        my $comment = $report->comments(undef, { rows => 1, order_by => { -desc => "id" }})->first;
+        isnt $comment->photo, '', 'photo added to comment';
```

Comment:
I think photo would be null if no photo was added, though, in which case this would pass (as only undef can match undef)? Probably best to check it has the hash you expect.